src/SourceBuild/content/source-mappings.json

```diff
@@ -12,6 +12,9 @@
     // These patches are applied on top of the code synchronized into the VMR
     "patchesPath": "src/installer/src/SourceBuild/patches",
 
+    // Location within the VMR where the source-mappings file is stored
+    "sourceMappingsPath": "src/installer/src/SourceBuild/content/source-mappings.json",
```

Comment:
Wouldn't the path in the VMR be at the root - `source-mappings.json`?

Reply:
Yes, we know. However, this PR is a bit outdated and will require [further changes](https://github.com/dotnet/arcade-services/pull/2167/files) before it goes in. We are trying to go around this problem of having the `source-mapping.json` in already